releases/v21.1.0-rc.2.md

```diff
@@ -0,0 +1,50 @@
+---
+title: What&#39;s New in v21.1.0-rc.2
+toc: true
+summary: Additions and changes in CockroachDB version v21.1.0-rc.2 since version v21.1.0-rc.1
+---
+
+## May 05, 2021
+
+Get future release notes emailed to you:
+
+{% include marketo.html %}
+
+### Downloads
+
+<div id="os-tabs" class="filters clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v21.1.0-rc.2.linux-amd64.tgz"><button id="linux" class="filter-button" data-scope="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v21.1.0-rc.2.darwin-10.9-amd64.tgz"><button id="mac" class="filter-button" data-scope="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v21.1.0-rc.2.windows-6.2-amd64.zip"><button id="windows" class="filter-button" data-scope="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v21.1.0-rc.2.src.tgz"><button id="source" class="filter-button" data-scope="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+<section class="filter-content" data-scope="windows">
+{% include windows_warning.md %}
+</section>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v21.1.0-rc.2
+~~~
+
+### Enterprise edition changes
+
+- [Changefeeds](../v21.1/change-data-capture.html) more reliably fail when [`IMPORT INTO`](../v21.1/import-into.html) is run against a targeted table. [#64372][#64372]
+
+### Bug fixes
+
+- Fixed a correctness bug, which caused partitioned index scans to omit rows where the value of the first index column was `NULL`. This bug was present since v19.2.0. [#64046][#64046]
+- [`IMPORT`](../v21.1/import.html) and [`RESTORE`](../v21.1/restore.html) jobs that were in progress during a cluster backup will now be canceled when that cluster backup is restored. This fixes a bug where these restored jobs may have assumed to make progress that was not captured in the backup. [#64352][#64352]
```

Comment:
```suggestion
- [`IMPORT`](../v21.1/import.html) and [`RESTORE`](../v21.1/restore.html) jobs that were in progress during a cluster backup will now be canceled when that cluster backup is restored. This fixes a bug where these restored jobs may have recorded prior progress that was not captured in the backup. [#64352][#64352]
```

Big caveat: Not sure if this is accurate as the original PR is a bit hard to read, however, I think this needed a touch more clarity (e.g. who is assuming and when) so please consider this edit *if* you think it's accurate.